README.md

```diff
@@ -1,60 +1,118 @@
+* `python scripts_generate/create_clf_data.py` Create images for classification training 
+* `python scripts_tf/create_tf_records.py --image_dir ./scripts_generate/data --output_dir ./model_data` Reformat training files
 
-Now the tests can be run by simply calling:
+### Training Pre-Classifier
+```
+python scripts_tf/train_clf.py \
+    --model_name MODEL_NAME \
+    --train_dir models/MODEL_NAME/checkpoints \
+    --records_name model_data/records
+```
+To evaluate the model's accuracy during training, run:
+```
+python scripts_tf/eval_clf.py \ 
+    --model_name MODEL_NAME \
+    --checkpoint_path models/MODEL_NAME/checkpoints \
+    --eval_dir models/MODEL_NAME/checkpoints/eval
+```
+Training statistics can be visualized with `tensorboard --logdir models/MODEL_NAME/checkpoints`.
 
-```sh
-$ tox
+### Freeze Pre-Classifier 
+After training, freeze the classification model for inference.
+```
+python scripts_tf/freeze_clf.py \
+    --model_name inception_v3 \
+    --ckpt_dir models/MODEL_NAME/ckpts \
+    --output_dir models/MODEL_NAME/frozen 
 ```
 
-This takes care of installing the development dependencies in a Python virtual
-environment. After, it runs quick unit tests to ensure the model
-file is loading as expected.
+### Optimize Pre-Classifier from ckpt
+```
+python scripts_tf/optimize_clf.py \
+    --input_saved_model_dir models/MODEL_NAME/frozen  \
+    --output_saved_model_dir models/MODEL_NAME/optimized \
+    --data_dir model_data/records \
+    --mode validation \
+    --use_trt \
+    --precision FP32 \
+    --batch_size 5
+```
 
-To only build the shapes and run the training step, or run the unit tests, you
-can run `tox -e model` and `tox -e unit`, respectively.
+### Training Object Detector Model
 
-These tests are automatically run on CircleCI on each commit.
+1. In a seperate folder `git clone https://github.com/tensorflow/models.git`
+2. Run with `MODEL_NAME` set to one of the models in `models/`
+```
+python path/to/models/research/object_detection/model_main.py \
+    --pipeline_config_path models/MODEL_NAME/pipeline.config \
+    --model_dir models/MODEL_NAME \
+    --num_train_steps 5000 \
+    --sample_1_of_n_eval_examples 1 \
+    --alsologtostderr
+```
+### Freeze Model Object Detector Model
+This will create a `frozen_inference_graph.pb` that will be optimized with tensorrt.
+```
+python path/to/models/research/object_detection/export_inference_graph.py \
+    --input_type=image_tensor \
+    --pipeline_config_path=models/MODEL_NAME/pipeline.config \
+    --trained_checkpoint_prefix=models/MODEL_NAME/model.ckpt \
+    --output_directory=models/MODEL_NAME/frozen
+```
+### Optimize Object Detector Model
+```
+python scripts_tf/optimize_od.py \
+    --input_saved_model_dir=models/MODEL_NAME/saved_model \
+    --output_saved_model_dir=models/MODEL_NAME/optimized \
+    --data_dir model_data/records/ \
+    --use_trt \
+    --precision FP32
+```
```

Comment:
In the future it prob be helpful to just build a cli do automate these tedious cmds. Good for now thou.

Reply:
Agreed. I'll make a issue on the projects for this semester.